app/templates/navbar.html

```diff
@@ -1,15 +1,10 @@
 <header class="navbar">
-  <img
-    class="logo"
-    src="{{ url_for('static', filename='/images/NextStep-logo.webp') }}"
```

Comment:
do you mean to switch back from the url_for function to string filepath? would recommend keeping the function in place, tho it's most handy when referencing a blueprint path.